pkg/event/config.go

```diff
@@ -278,8 +279,9 @@ func ParseConfig(reader io.Reader, region string, targetList *TargetList) (*Conf
 		return nil, err
 	}
 
+	// Empty queue list means user wants to delete the notification configuration.
 	if len(config.QueueList) == 0 {
-		return nil, errors.New("missing queue configuration(s)")
+		return &config, nil
```

Comment:
Remove this `if` block as this check is already handled in xml unmarshaling.

Reply:
done